imports/ui/case/case-search.actions.js

```diff
@@ -1,11 +1,14 @@
 export const UPDATE_SEARCH_RESULT = 'update_search_result'
 export const FINISH_SEARCH = 'finish_search'
 export const START_SEARCH = 'start_search'
+export const NAVIGATION_REQUESTED = 'navigation_requested'
+export const NAVIGATION_GRANTED = 'navigation_granted'
 
 export function updateSearch (searchText) {
   return {
     type: UPDATE_SEARCH_RESULT,
-    searchText: searchText
+    searchText: searchText,
+    navigationRequested: false
```

Comment:
`navigationRequested` doesn't need to be included in the action's payload as its value is already implied by the type of action and should be resolved by the reducer. The actions should carry as little "decisions" as possible

Reply:
The reducer is currently not using the value from the actions anyway. These extra attributes (navigationRequested) can be dropped just in this file.